readme.md

```diff
@@ -37,7 +42,9 @@ Your Answer:
 Describe the differences between a SQL and NoSQL DB, and when you might use each.
 
 Your Answer:
-```text
+```SQL we have a schema, but in order to manipulate that schema we have to re-seed our data to
```

Comment:
You are correct that SQL DBs have a schema, and NoSQL DBs generally don't. But some of your answer was a bit off base. For example:
* We don't need to re-seed to update a SQL schema. You can add / rename, columns and tables without losing any data.
* You referenced Mongoose as a NoSQL DB, but it's not, it's an ORM (like ActiveRecord is for PSQL). Mongo is the DB.
* Looking for more on the *why*, check the model answer on the solution branch for an example.